CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [3.0.0-rc2] - 2020-08-26
+
 ### Added
 - Support for using `Deferred` secrets fetching via `conjur::secret`.
   [cyberark/conjur-puppet#13](https://github.com/cyberark/conjur-puppet/issues/13).
```

Comment:
Can you be more specific here and for the PDK line below?  I think that we should ban use of the word "support" from CHANGELOGs.  Something more like:

> Secrets are now retrieved by the Puppet agent, instead of the server, using Puppet's deferred function feature.  This aligns with Puppet's documented best practices.

Regarding PDK, what is the benefit?